bin/generate-samples.sh

```diff
@@ -7,7 +7,7 @@ declare root="$(cd "$cwd" && cd ../ && pwd)"
 declare executable="${root}/modules/openapi-generator-cli/target/openapi-generator-cli.jar"
 
 if [ ! -f "$executable" ]; then
-  (cd "${root}" && mvn -B --no-snapshot-updates clean package -DskipTests=true -Dmaven.javadoc.skip=true -Djacoco.skip=true)
+  (cd "${root}" && ./mvnw -B --no-snapshot-updates clean package -DskipTests=true -Dmaven.javadoc.skip=true -Djacoco.skip=true)
```

Comment:
I don't think this should be changing - `./mvnw` is path relative

Reply:
Yes but we always are in root (`cd ${root}`) so it's always relative to that. I'm probably not the only one without a globaly installed maven binary that can't execute this script without this change.

If you insist, il will remove this change.

Reply:
@alucas yes, please revert

Reply:
done